rd_workshop-2/post.js

```diff
@@ -0,0 +1,37 @@
+let linkUserPost = document.querySelector("#post-box");
+let dataUser = JSON.parse(localStorage.getItem('dataUser'));
+fetch('https://gorest.co.in/public/v2/posts')
```

Comment:
there is separated API to get posts of user:
`https://gorest.co.in/public/v2/users/[userId]/posts`

you had to use it instead of filtering ALL posts